cardano-node/src/Cardano/Node/Protocol/Cardano.hs

```diff
@@ -85,12 +85,6 @@ mkSomeConsensusProtocolCardano NodeByronProtocolConfiguration {
                              npcAlonzoGenesisFileHash
                            }
                            NodeHardForkProtocolConfiguration {
-                            -- npcTestEnableDevelopmentHardForkEras,
```

Comment:
So this comment is still valid. We only need to reflect the name change in the comment.

Reply:
@LudvikGalois this is not resolved. The comment is still deleted.

Reply:
Because the field has been renamed. A new comment was added

Reply:
Where is the new comment?

Reply:
On line 88, where the old one was https://github.com/input-output-hk/cardano-node/pull/4341/files#diff-8c649864fe4b96d142cfd4c2bb9e32a9510fe737c6ad4e13e04b585ee15a6b74R88

Reply:
The comment being deleted provides context as to what the flag is being used for. Without it newcomers will have a more difficult time understanding the purpose of the flag.

Reply:
`--npcTestEnableDevelopmentProtVer` is not a sufficient replacement for the comment being deleted. Please restore the comment.

Reply:
My mistake, I completely misunderstood what was meant by "the comment"